### ISSUE TITLE
Initial Requests are delayed ~5s 

### ISSUE THREAD

**Issue by dusty** [a85/POSTMan-Chrome-Extension#6073]
**Describe the bug**
Initial requests seem to take about 5s **before** they are sent out from postman.  Subsequent requests are fast.  After a short period of several seconds between requests the delay is reintroduced.

**To Reproduce**
1.  Send out a request and look at the response time
2.  Immediately send out another request and look at the response time
3.  Wait a few seconds and try again

**Expected behavior**
I would expect that all queries happen immediately.

**Screenshots**
In the recording at the link, I am sending out a simple request to a node app running on localhost.  You can see the delay is happening before the request makes it to the node app.  Notice the execution time in the app logs are consistently fast.

http://recordit.co/74CdgMHbSH

**Edited** I initially uploaded a gif, but it was cutoff pretty quickly.  Follow the link to the full video.

**App information (please complete the following information):**
 - Native App
 - Postman Version 6.7.4
 - macOS 10.14.3

**Additional context**
The 5000ms delay is consistent on any endpoint not just limited to my localhost node app I used for the demo.

There are no pre-request scripts assigned to this collection or to this query.

**Comment by codenirvana** [comment]
@dusty The subsequent requests are using a persistent connection.

Postman by default adds a `Connection: Keep-Alive` header which keeps the connection socket live until it times out. Try sending the request using `Connection: close` header and you will see similar response time for every request.

So, with [Keep-Alive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Keep-Alive) the subsequent request don't have to do the following steps:
- DNS Lookup
- TCP Handshake
- SSL/TLS Handshake

But, right now this detailed request-response timing information is missing in the Postman app and we already started working towards adding this support (shipped with [Newman](https://npmjs.com/package/newman) using the `--verbose` flag).

Closing this issue because this is expected behavior. Detailed request timing information will be available in the app very soon!

**Comment by dusty** [comment]
Thanks for the response.

If I change to connection: close I do see similar response times on every request.  

However, it still seems slow to me.  If I run just a simple curl command it responds immediately without any prior keep alives setup.

```
> time curl http://localhost:8080
real	0m0.018s
user	0m0.006s
sys	0m0.006s
```

What is interesting is that I looked at the logs in the shared console and I see a syncing with postman on each request.  These all post in the logs **before** it hits my local app.

```
sync client process worker started [Object]
shared.js:16757 history:import:xxxx
vendor-shared.js:123069 Sync Request: /v3/history/import?owner=427160&workspace=xxxx
vendor-shared.js:123073 Sync Request: Object {method: "post", url: "/v3/history/import?owner=xxxx&workspace=xxxx", headers: Object, data: Object}
vendor-shared.js:123074 Sync Response: Object {model_id: "xxxx", meta: Object, data: Object, revision: 6980725583, jwr: Object}
shared.js:17768 removing changesets from sync client [Object]
```

It looks like it is recording all the requests to the history.  However, could this be causing a delay?  Is it trying to complete this before it sends the request out?  I'm not certain if that is going up to the cloud or just posting to a local server running inside postman app to record it.

Either way, is it possible to disable that if its possibly the cause of the delay?  I'd rather have quicker tests than be able to view the history.

Thanks

**Comment by codenirvana** [comment]
@dusty That should not affect the ongoing request. Also, I am not very sure that `time` calculates the actual request time.

I would suggest trying the following script for calculating response time:
```console
$ curl -o /dev/null -s -w 'Total: %{time_total}s\n' https://example.com
```

**Comment by dusty** [comment]
Thanks for the curl tip and your patience. 

I figured it out and its due to OSX and its handling of .local domains.  I was pointing my queries in postman to a .local address, which I had in /etc/hosts pointing to 127.0.0.1.

The mistake I made was testing it via curl and the browser using localhost or 127.0.0.1, so I never came across the issue.  

On OSX if you point to a .local domain it treats the address as a Bonjour address.  I'm not sure exactly how that works, but if you don't have a matching IPv6 address in /etc/hosts then it does something else first before it falls back to the IPv4 address in /etc/hosts.

So, by adding both IPv4 and IPv6 to /etc/hosts, the problem resolves.
```
127.0.0.1 test123.local
::1 test123.local
```

Apparently its only the .local domain that this causes issues on.

<img width="970" alt="Screen Shot 2019-03-21 at 9 36 15 AM" src="https://user-images.githubusercontent.com/792/54768937-8263ee00-4bbd-11e9-8564-a387b81fbe38.png">
